articles/112_generated_interfaces_cpp.md

```diff
@@ -104,6 +104,13 @@ All other constants are declared as `static const` members in the struct and the
 ### Constructors
 
 The *default constructor* initializes all members with their default value.
+In some cases this may not be desirable, since these fields will often be immediately overwritten with user-provided values.
+Therefore, the constructor takes an optional directive of type `enum rcl_msg_init_type` to control how initialization is done:
```

Comment:
Right, understood.  Thanks for the explanation.  Given that we want this to be from the C layer, what do you think about a new package `rcl_core`?  At this point it would just contain a single header file with this enum, we could expand it over time.  Other ideas for this welcome.